pages/hackathons.tsx

```diff
@@ -56,11 +63,16 @@ export default function Hackathons() {
                 setError(arr)
             })
     }, [error])
+    const [selectedTab, setSelectedTab] = useState(tab)
+    const handleSelect = (eventKey) => {
+        setSelectedTab(eventKey)
+        router.push(`?tab=${eventKey}`, undefined, { shallow: true })
+    }
 
     return (
         <div style={{ background: "#87a3bb17", minHeight: "100vh" }}>
             <Head>
-                <title>Hackathons | COPS Hackalog</title>
+                <title>{`${selectedTab} Hackathons | COPS Hackalog`}</title>
```

Comment:
The lowercase `selectedTab` looks a bit awkward. As a simple fix, we can change the keys from 'ongoing' to "Ongoing".

Reply:
Change event key from `ongoing` to `Ongoing`. Similarly for `completed` and `upcoming`.
```jsx
<Nav.Link eventKey="ongoing">Ongoing</Nav.Link>
```
```jsx
<Tab.Pane eventKey="ongoing">....</Tab.Pane>
```